server_addon/hiero/server/settings/publish_plugins.py

```diff
@@ -27,11 +27,21 @@ def validate_name(cls, value):
         return normalize_name(value)
 
 
+class CollectClipEffectsTracksModel(BaseSettingsModel):
+    _layout = "expanded"
+    name: str = SettingsField("", title="Name")
+    track_regex: str = SettingsField("", title="Track Regex")
```

Comment:
this should be different type > list[str] 

There will be cases where we will need to hove node classess from multiple tracks to be one product.

Reply:
We can target multiple tracks with a single regex. But if we are going down the route of multiple track names, then we might as well be explicit and not use regex, so something like `track_names` list where people can input `creative_look` and `invert_look` for example.

Reply:
yeah lets  just be explicit, then. Important is that there will be multiple track inputs.

Reply:
Done